nncf/onnx/quantization/quantize_model.py

```diff
@@ -9,17 +9,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import sys
+from pathlib import Path
 from typing import Any, Callable, Iterable, Optional, TypeVar, Union
 
 import onnx
+from onnx.external_data_helper import ExternalDataInfo
+from onnx.external_data_helper import _get_all_tensors
```

Comment:
Please, do not use private functions.

Reply:
Yes, you are right. But ONNX doesn't provide an API to properly handle all tensors. So, it is a trade-off between private functionality and copying all that functionality into our repo. I'd be happy to copy it if you'd like.